docs/dgin10mins.adoc

```diff
@@ -7,89 +7,74 @@ image::dg8.png[Red Hat Data Grid 8.0, 700]
+So the most straightforward way of handling this would be to store some of that data in the memory. By doing so, you would be able to process requests to your systems much faster. However there are challenges. How much data should you store in memory? And most importantly, what will happen in case of failure scenarios? 
 
-- Will you lose all the state of your data in memory? 
-- Will you need to re-read all your data and events inorder to get back to the same state where you failed. Or you might have to let go of that entirely. 
+* Will you lose all the state of your data in memory? 
+* Will you need to re-read all your data and events inorder to get back to the same state where you failed. Or you might have to let go of that entirely. 
 
-Now the above two might seem very simple, but that tasks can be tideous and most importantly error prone. 
-So at this point we could introduce a local cache that e.g. could be a ConcurrentHashMap. and alot of us might have done this in the past. However as a developer you might know that this has not much effect in case of failures and the handling that you will need to do incase of failures. 
+Now the above two might seem very simple, but that tasks can be tedious and most importantly error prone. 
+So at this point we could introduce a local cache (e.g. a `HashMap`) that a lot of us might have done this in the past. However as a developer you might know that this has not much effect in case of failures. 
 So the need is entirely for a component that can not just cache data in the memory, but give 
 
-1. A consistent way to handle data and state in the memory. 
-2. Resiliencey in case of failures. 
-3. Processing efficency and performance.
-4. Events, streams, and distribution capabilites. 
+1. A consistent way to handle data and state in the memory
+2. Resiliencey in case of failures
+3. Processing efficency and performance
+4. Events, streams, and distribution capabilites
 
 
 image::caching.png[Caching, 700]
 
 
-By having such capabilites a cache is no longer just a datastructure in the memory, but also as a developer now you have the possiblity to take this component out of your local in memory processing and distribute it out on the network. Thereby incase of application failures you will still be able to access this data from the last point where you left off. 
+By having such capabilites a cache is no longer just a data structure in memory, but also as a developer now you have the possiblity to take this component out of your local in memory processing and distribute it out on the network. Thereby in case of application failures you will still be able to access this data from the last point where you left off. 
```

Comment:
suggestion: cache is no longer just an in memory data structure, ..

Reply:
Pushed a new commit based on your suggestion.